velox/common/base/VeloxException.h

```diff
@@ -144,6 +149,9 @@ inline constexpr auto kUnknown = "UNKNOWN"_fs;
 
 class VeloxException : public std::exception {
  public:
+  /// Define the velox exception type.
```

Comment:
nit: This comment doesn't seem to provide any additional information to what's available from the class and enum names. Consider elaborating or removing.